dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/LocalFileTreeTablePresenter.java

```diff
@@ -277,6 +278,11 @@ private void transferToBlackPearl() {
                 alert.showAlert(resourceBundle.getString("fileSelect"));
                 return;
             }
+            for (final TreeItem<FileTreeModel> fileTreeModelTreeItem : currentSelection) {
```

Comment:
this could be cleaner with currentSelection.stream().filter(ftmItem -> !Files.isReadable(ftmItem.getValue().getPath()).forEach( unreadableFtmItem -> loggingService::logMessage("message...")